PROXY.md

```diff
@@ -1,61 +1,68 @@
 # The Athens Proxy
 
-Athens comes with a module proxy that you can run inside your own organization.
-This proxy will do two things for you:
+The Athens project has two components, the [central registry](./REGISTRY.md) and edge proxies.
+This document details the latter.
 
-* Cache modules locally, so that developers inside your organization do not
-  have to fetch modules from the public internet
-* Redirect module fetch operations to a specific location on the internet
+# The Role of the Proxy
 
-It's intended that clients set the `GOPROXY` environment variable on their
-machine to the Athens proxy address before they start developing. That
-enables the following:
+We intend proxies to be deployed primarily inside of enterprises to:
 
-```console
-go get github.com/a/b
-```
+- Host private modules
+- Exclude access to public modules
+- Cache public modules
 
-This command will go fetch the `github.com/a/b` module from the proxy,
-instead of the VCS.
+Importantly, a proxy is not intended to be a complete _mirror_ of an upstream registry. For public modules, its role is to cache and provide access control.
 
-# Proxy Configuration
+# Proxy Details
 
-TODO
+First and foremost, a proxy exposes the same vgo download protocol as the registry. Since it doesn't have the multi-cloud requirements as the registry does, it supports simpler backend data storage mechanisms. We plan to release a proxy with several backends including:
 
-# Proxy Caches
+- In-memory
+- Disk
+- RDBMS
+- Cloud blob storage
 
-The proxy can be configured to store cached modules in several places:
+Users who want to target a proxy configure their `vgo` CLI to point to the proxy, and then execute commands as normal.
```

Comment:
Should we mention that the configuration is through `GOPROXY` env var? Just in case people wanna test it out

Reply:
I was thinking of creating a separate section / document for how to test the thing (probably in `DEVELOPMENT.md`) and linking to it from here

Reply:
I think that's a good idea